### PR TITLE
Handle raw Airbyte records in Faros destination + more GH converters

### PR DESCRIPTION
**Pull request by tovbinm**
## Description
1. Handle raw Airbyte records in Faros destination. This enables Faros destination to handle inputs in two scenarios:
- GitHub source -> Faros destination
- GitHub source -> Postgres destination (raw), and then Postgres source (raw) -> Faros destination

2. Added more GitHub converters

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

### CODE REVIEW

File: cdk/src/protocol.ts [hunk 1]
```diff
@@ -128,6 +129,18 @@ export class AirbyteRecord implements AirbyteMessage {
     }
   ) {}
 
+  unpackRaw(): AirbyteRecord {
+    const stream = this.record.stream;
+    if (!stream || !stream.startsWith('_airbyte_raw_')) {
```
Comment (cjwooo) [inline cdk/src/protocol.ts:134]: Make a global const for `_airbyte_raw_`

File: destinations/faros-destination/src/converters/github/issues.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import {AirbyteRecord} from 'cdk';
+import {Utils} from 'faros-feeds-sdk/lib';
```
Comment (cjwooo) [inline destinations/faros-destination/src/converters/github/issues.ts:2]: Why is the `/lib` there?

Comment (tovbinm) [reply destinations/faros-destination/src/converters/github/issues.ts:2]: removed

File: cdk/src/protocol.ts (hunk 1 above)
Comment (tovbinm) [reply cdk/src/protocol.ts:134]: will do

**Review by cjwooo — APPROVED** [review]

**Pull request merged**